src/Microsoft.PowerShell.Commands.Utility/resources/TestJsonCmdletStrings.resx

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
+    Microsoft ResX Schema 
+    
```

Comment:
Please revert all extra whitespaces.

Reply:
I didn't (explicitly) add those.  That's a file managed by VS.

Reply:
There is an option to trim tail spaces.

Reply:
<img width="756" alt="image" src="https://user-images.githubusercontent.com/2676804/217669917-1e98d92b-b5c3-4661-8178-859bd64c740b.png">
This is the interface I'm given.  I don't interact with the XML.

Regarding VS settings, I haven't changed anything in VS, and I don't want to set anything that's going to affect _all_ of my other projects as I'm operating across multiple repos with different rules regarding this.

Reply:
It doesn't matter how you get rid of this garbage. You can edit the original file even in Notepad.

Reply:
I've done it, but being required to edit a generated file is a bit hacky.